jihyeon/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER SHOULD BE HERE!
+
+DB_HOST= host #
+DB_USER=root
+DB_PASS=password
+DB_CONNECTION = mysql
+DB_DATABASE = database
+DB_PORT = 3306
+DB_LOGGING = TRUE
```

```suggestion
PORT = YOUR PORT NUMBER SHOULD BE HERE!

DB_HOST= host #
DB_USER=db_user
DB_PASSWORD=password
DB_CONNECTION = rdbms_type
DB_DATABASE = database
DB_PORT = db_port
DB_LOGGING = true / false
```
1. root 와같은 database user name을 중의적으로 표현해주세요.
2. DB_PASSWORD라고 명확히 명칭해주세요!
3. 어떤 rdbms를 쓰는지 해당 rdbms의 port number는 무엇인지 모두 직접 명시하지 않고 중의적으로 표현해주세요.

---

dbmate의 타겟 주소 목적의 `DATABASE_URL` 변수 또한 중의적인 표현을 활용하여 작성해주세요!
```jsx
// Example	

DATABASE_URL="rdbmsType://username:password@iphost:portNumber/databaseName"
```